managed-ledger/pom.xml

```diff
@@ -66,6 +66,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
It's a little confusing that add the pulsar-client-original dependency to managed-ledger.
Is there anyway we can move the dependent part to pulsar-common? Since we did not add client to managed-ledger right?

Reply:
Yes, we can add this dependency to managed ledger

Reply:
Thanks for your suggestion. I've move the dependency part from pulsar-client-original to pulsar-common